### PR TITLE
Simplify linter script

### PR DESCRIPTION
**Pull request by sturivny**
As we don't need to use token anymore, simplified scripts.

### CODE REVIEW

File: tests/pipeline_linter.sh [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-    esac
-  done
+if [ $# -ne 1 ]; then
+    echo $0: "USAGE: ./pipeline_linter.sh jenkins_filename"
+    exit 1
+fi
+
+FILE=$1
 
 OUTPUT=$(curl -H "Authorization: OAuth " -X POST -F "jenkinsfile=<$FILE" https://osci-jenkins-2.ci.fedoraproject.org/pipeline-model-converter/validate)
```
Comment (bgoncalv) [inline tests/pipeline_linter.sh:10]: Is `-H "Authorization: OAuth "` still needed?

Comment (sturivny) [reply tests/pipeline_linter.sh:10]: If I remove it, will be `not pretty` ouput

Comment (bgoncalv) [reply tests/pipeline_linter.sh:10]: okay, so just leave it there :)

**Comment by bgoncalv** [comment]
lgtm

**Review by bgoncalv — APPROVED** [review]

**Pull request merged**